lib/cucumber/cli/options.rb

```diff
@@ -334,12 +341,10 @@ def require_jars(jars)
       def set_language(lang)
         require 'gherkin/dialect'
 
-        if lang == 'help'
-          list_languages_and_exit
-        elsif !::Gherkin::DIALECTS.keys.include? lang
-          indicate_invalid_language_and_exit(lang)
-        else
+        if ::Gherkin::DIALECTS.keys.include? lang
```

Comment:
Now that this method is shorter, could you swap this logic around round to use a [Guard Clause](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html)?

i.e.
```
return indicate_invalid_language_and_exit(lang) if !::Gherkin::DIALECTS.keys.include? lang
list_keywords_and_exit(lang)
```